### PR TITLE
fix: Hungarian translation

### PR DESCRIPTION
**Pull request by KallosLaszlo**
Corrections and additions to the Hungarian translation

Also includes the latest chopper patch. (According to the old description, it was not supported.)

### CODE REVIEW

File: translations/translation_hu.xml [hunk 1]
```diff
@@ -603,295 +603,308 @@ A szerkesztő mód aktiválása a következő opciókat teszi elérhetővé:
...
+A kombájn kirakásához használhat normál pótkocsit vagy áttöltő kocsit.
+A pótkocsik ürítéséhez használhatja a Giants Helpert (FS22 Segítő) vagy az AutoDrive-ot.
 
-Unloaders have only a few settings you can tweak:
-- You can set the fill level at which the unloader will consider itself full. This
-is to avoid waiting for a combine to call with a trailer almost full.
-- You can use the offsets in the HUD to adjust the position of the unloader under the harvester's pipe.
+A kirakó elindításához néhány dologra van szükség:
+- Egyetlen utánfutót vagy áttöltő kocsit kell csatlakoztatni. Több utánfutó nem támogatott.
+- Használja a HUD Cél ikonját a betakarított tábla kiválasztásához. A kirakodó kirakja az összes Courseplay hajtott betakarítógépet, amely azon a területen dolgozik.
```
Comment (pvaiko) [inline translations/translation_hu.xml:809]: Courseplay segítő által vezetett

File: translations/translation_hu.xml [hunk 2]
```diff
@@ -603,295 +603,308 @@ A szerkesztő mód aktiválása a következő opciókat teszi elérhetővé:
...
+- A kirakodó elindításához a pályán vagy a pálya 20 méteres körzetében kell lennie.
```
Comment (pvaiko) [inline translations/translation_hu.xml:810]: a táblán, vagy annak 20 méteres körzetében

File: translations/translation_hu.xml [hunk 3]
```diff
@@ -603,295 +603,308 @@ A szerkesztő mód aktiválása a következő opciókat teszi elérhetővé:
...
 
+A kirakodó csak néhány beállítással rendelkezik, amelyeket módosíthat:
+- Beállíthatja azt a feltöltési szintet, amelynél a kirakodó telinek tekinti magát. Ez
+Az, hogy ne várjon egy kombájnra, ha a pótkocsi majdnem tele van.
```
Comment (pvaiko) [inline translations/translation_hu.xml:814]: Ez Az,?

File: translations/translation_hu.xml [hunk 4]
```diff
@@ -603,295 +603,308 @@ A szerkesztő mód aktiválása a következő opciókat teszi elérhetővé:
...
+- A HUD-ban található eltolásokkal beállíthatja a kirakodó helyzetét a betakarítógép csöve alatt.
 "/>
-    <text name="CP_help_page_unloaderTrailer_title" text="Using trailers"/>
+    <text name="CP_help_page_unloaderTrailer_title" text="Utánfutók használata"/>
```
Comment (pvaiko) [inline translations/translation_hu.xml:817]: Pótkocsi jobban hangzik, legyen mindenhol az

File: translations/translation_hu.xml [hunk 5]
```diff
@@ -603,295 +603,308 @@ A szerkesztő mód aktiválása a következő opciókat teszi elérhetővé:
...
-If you use a trailer to unload, there are two ways to transport your harvest to a silo or to a selling point: the Giants helper
-or AutoDrive.
-You can either use the AI Menu and create a Job, then by activating the Giants helper function and unload target at the
-bottom of the list, or by activation of the wheel icon on the HUD.
-In the latter case, you have to choose the unload station after the wheel icon turns blue.
+Ha pótkocsit használsz a kirakodáshoz, kétféleképpen szállíthatja a termést a silóba vagy az értékesítési pontra: a Giants segítő (FS22 Segítő)
+vagy AutoDrive.
+Használhatja az AI menüt és létrehozhat egy munkát, majd aktiválhatja a Giants segítő funkciót, és kirakhatja a célt a
+a lista aljára, vagy a HUD kerék ikonjának aktiválásával.
+Utóbbi esetben a kerék ikon kékre váltásd, után kell kiválasztani a kirakodó állomást.
```
Comment (pvaiko) [inline translations/translation_hu.xml:823]: elgépelések

File: translations/translation_hu.xml [hunk 6]
```diff
@@ -603,295 +603,308 @@ A szerkesztő mód aktiválása a következő opciókat teszi elérhetővé:
...
-Just like with trailers, you use the target icon to choose the field where you want to unload a combine or harvester.
-To start the unloader, place it again on the field or within 20 meters of the field.
-If the auger wagon is full, it automatically looks for a trailer on or near the field to overload into it.
-In case the trailer is full and the auger wagon still not empty, it will look for another trailer to unload to.
-Is no trailer available, it will continue to unload the combine.
-"/>
-    <text name="CP_help_page_unloaderFieldUnload_title" text="Unload on the field"/>
+Az áttöltő kocsi használata nagyon egyszerű.
+Csakúgy, mint a pótkocsik esetében, itt is a célkereszt ikon segítségével választhatod ki azt a táblát, ahová ki szeretné rakni a kombájnt vagy a betakarítógépet.
+A kirakodó elindításához helyezze vissza az útvonalra vagy 20 méteren belülre.
```
Comment (pvaiko) [inline translations/translation_hu.xml:831]: a táblára vagy annak 20 méteres körzetébe

**Review by pvaiko — CHANGES_REQUESTED** [review]
Thanks for the translation, we really appreciate it, especially the help texts. I have a few comments, mainly regarding typos and consistent language. 

Thank you

**Comment by KallosLaszlo** [comment]
I really appreciate you taking the time to check it. I'll review the translation again and make the necessary corrections along with your suggestions.

**Comment by KallosLaszlo** [comment]
@pvaiko 
I have two questions. What Hungarian translation would you provide for the terms 'Giants helper' and 'multitool'?

**Comment by pvaiko** [comment]
> @pvaiko I have two questions. What Hungarian translation would you provide for the terms 'Giants helper' and 'multitool'?

Beépített segítő? Normál segítő? Don't really have a good idea for multitool either. Többgépes?

**Comment by pvaiko** [comment]
Are you still working on this, or just forgot to mark it ready for review?

**Comment by KallosLaszlo** [comment]
> Are you still working on this, or just forgot to mark it ready for review?

I will finish the work this weekend.

**Pull request merged**